scripts/webpack-esm.js

```diff
@@ -4,7 +4,7 @@ const path = require("path");
 const packageJSON = require("../package.json");
 
 const fs = require("fs");
-const componentDirectories = ["button", "mobile/checkbox","base/mobile-label", "base/mobile-error", "checkbox", "base/error", "base/label"];
+const componentDirectories = ["button", "mobile/checkbox","base/mobile-label", "base/mobile-error", "checkbox", "base/error", "base/label", "textarea"];
```

Comment:
There is a conflict with the latest version of "v1.4.0_SSR-1845_Version-conflicts".
Please merge the latest version of "v1.4.0_SSR-1845_Version-conflicts" into your branch.
Pay attention to the component order in "componentDirectories", thank you.